analyses/country_template/docs/style.css

```diff
@@ -1,11 +1,34 @@
-body{
+body {
   font-family: 'Source Sans Pro';
 }
 /* Headers */
-h1{
+h1 {
     font-family: Arvo;
     font-weight: bold;
     color: #1bb580;
+    font-size: 18px;
+}
+
+h2 {
+    font-family: Arvo;
+    font-weight:
```

Comment:
Empty CSS styling is preventing the color from being read properly below.

Reply:
Yesss! Thanks for the fresh eyes, Seth!